test/runtests.jl

```diff
@@ -3,7 +3,7 @@ using Test
 using StatsBase
 using Random
 
-a = rand(1:10000, 1000)
+a = rand(1:10000, 2200)
```

Comment:
Maybe make this a loop over arrays with the two sizes to cover both code paths?

Reply:
Done. 6a9b22639f9b522c32f8dd201db27cfa3e7c6b3a